### ISSUE TITLE
is it possible to have the times of the event to the right of the title?

### ISSUE THREAD

**Issue by nickbeaulieu98** [jclarke0000/MMM-MyCalendar#12]
I'd like the times of the events to show on the right of the title,

**Comment by zapperiopa** [comment]
Yes possible, in the MMM-MyCalendar.css, change the the display value to inline for 
MMM-MyCalendar .time
MMM-MyCalendar .location 
Split them up if you don't want to do it for location.
